src/test/HelmetTest.js

```diff
@@ -217,7 +217,20 @@ describe("Helmet", () => {
 
                 const htmlTag = document.getElementsByTagName("html")[0];
 
-                expect(htmlTag.attributes.length).to.equal(0);
+                expect(htmlTag.getAttribute("lang")).to.be.null;
+                expect(htmlTag.getAttribute("amp")).to.be.null;
```

Comment:
There are now 3 test cases around attributes initialized outside of Helmet:
1. At first, they cannot be cleared by Helmet
2. They can be overwritten by Helmet
3. Once they have been overwritten in Helmet, they can now be cleared as well